packages/lumx-react/src/components/slideshow/Slideshow.tsx

```diff
@@ -182,18 +182,29 @@ export const Slideshow: Comp<SlideshowProps, HTMLDivElement> = forwardRef((props
         onChange(currentIndex);
     }, [currentIndex, onChange]);
 
+    const slideshowSlidesId = 'slideshow-slides';
+
     /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
     return (
-        <div
+        <section
             ref={mergeRefs(ref, setElement)}
             {...forwardedProps}
             className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, theme }), {
                 [`${CLASSNAME}--fill-height`]: fillHeight,
                 [`${CLASSNAME}--group-by-${groupBy}`]: Boolean(groupBy),
             })}
-            tabIndex={0}
+            aria-roledescription="carousel"
+            aria-live={isAutoPlaying ? 'off' : 'polite'}
         >
-            <div className={`${CLASSNAME}__slides`}>
+            <div
+                id={slideshowSlidesId}
+                className={`${CLASSNAME}__slides`}
+                onMouseEnter={() => setIsAutoPlaying(false)}
+                onMouseLeave={() => setIsAutoPlaying(Boolean(autoPlay))}
+                onBlur={() => setIsAutoPlaying(Boolean(autoPlay))}
+                onFocus={() => setIsAutoPlaying(false)}
+                tabIndex={0}
```

Comment:
I don't think the whole slide should be focusable by default.
It should only be focusable if it is interactive (button / link).

Maybe we could "simply" let the child decide of this behavior ? 
Don't set any tabIndex at all and whatever is set inside the slideshow will decide if it should be focusable or not.